README.md

```diff
@@ -113,7 +115,27 @@ resource "cloudflare_record" "terraform_managed_resource" {
   balancers, spectrum applications, etc)
 * A valid Cloudflare API key and sufficient permissions to access the resources
   you are requesting via the API
-* An initialised Terrafrom directory
+* An initialised Terraform directory
+
+### Terraform directory
+
+To set up a terraform directory you need a config file `main.tf` containing
+
+```
+terraform {
+    required_providers {
+        cloudflare = {
+            source = "cloudflare/cloudflare"
+        }	
+    }
+}
+
+provider "cloudflare" {
+# api_token  = ""  ## Commented out as we are using an environment var
+}
+```
+
+You then need to run `terraform init` within this directory to initialize it.
```

Comment:
this isn't really the place for telling people _how_ to organise and run Terraform; instead, let's just call out the command explicitly.

```suggestion
* An initialised Terraform directory (`terraform init` has run and providers installed)
```